### PR TITLE
fix: creating bob.json for nightly

### LINKED ISSUE #107: Error: Couldn't find bob.json
First off, thanks for making this tool!

After the recent update that addresses `nightly` not being updated, I'm getting this error when running `bob use nightly`.
The output of the command is:
```
Mar 13 01:19:18.878  INFO Looking for nightly updates
Mar 13 01:19:18.957 ERROR Error: Couldn't find bob.json
```
`bob list` output:
```
┌───────────┬─────────────┐
│  Version  │  Status     │
├───────────┼─────────────┤
│  nightly  │  Installed  │
└───────────┴─────────────┘
```
I have tried `bob erase` and reinstalled `nightly`, it installs fine but errors when trying to use it. I'm on version `2.1.2`

### PR DESCRIPTION
**Pull request by utkarshgupta137**
Fixes #107.

### CODE REVIEW

File: src/handlers/install_handler.rs [hunk 1]
```diff
@@ -29,34 +29,33 @@ pub async fn start(
...
-        info!("Looking for nightly updates");
-
-        nightly_version = Some(helpers::version::nightly::get_upstream_nightly(client).await?);
-        let upstream_nightly = nightly_version.as_ref().unwrap();
-        let local_nightly = helpers::version::nightly::get_local_nightly(config).await?;
-
-        if upstream_nightly.published_at == local_nightly.published_at {
-            return Ok(InstallResult::NightlyIsUpdated);
-        }
+    let nightly_version = if version.version_type == VersionType::Nightly {
```
Comment (MordechaiHadad) [inline src/handlers/install_handler.rs:32]: Why are you reverting back to the old way I handled nightly? I changed it cuz it had too much indentation and looked pretty ugly

Comment (utkarshgupta137) [reply src/handlers/install_handler.rs:36]: Fixed

**Comment by MordechaiHadad** [comment]
Thank you looks good! I am gonna wait until the original issue poster tries this PR or I'll merge it tomorrow morning which ever comes first.

**Comment by MordechaiHadad** [comment]
Sorry for the delay, and thank you! Your contributions really helped me!

**Pull request merged**